scripts/globals/mobskills/gnash.lua

```diff
@@ -0,0 +1,31 @@
+-----------------------------------
+-- Gnash
+-- Deals damage equal to 1/2 player's HP
+-- Need confirmation on whether this is 1/2 player's total HP
+-- or 1/2 player's current HP.
```

Comment:
I think we should add all of these to the module as well to keep it updated for future LSB prs